shell/platform/darwin/macos/framework/Source/FlutterEngine.mm

```diff
@@ -679,9 +688,7 @@ - (void)shutDownEngine {
     return;
   }
 
-  if (_viewController && _viewController.flutterView) {
-    [_viewController.flutterView shutdown];
-  }
+  [self.viewController.flutterView shutdown];
```

Comment:
I may be missing something, but won't `viewController` be nil in headless mode here?

Reply:
![image](https://user-images.githubusercontent.com/737941/213535673-24bbd108-fb53-4f3f-a93f-7d383ce3213e.png)

Reply:
Yeah, this feature has screwed me up more than I like it. But it exists, so...